asciidoc/06-modelling-answers.adoc

```diff
@@ -817,13 +816,13 @@ weighing less than fifteen pounds and
 whose length plus girth exceeds seven feet
 are chargeable at the fifteen pound rate.
 
-Size Standards&colon;:: Items whose thickness is less than 0.007 inches are not
+Size Standards&#58;:: Items whose thickness is less than 0.007 inches are not
 accepted for delivery. Items less than 0.25
 inches in thickness must be rectangular
 in shape and at least 3.5 inches high and at least 5 inches long. Items
 weighing more than 70 pounds are not accepted for delivery.
 
-Environmental&colon;:: Neither hazardous materials nor live or dead animals
+Environmental&#58;:: Neither hazardous materials nor live or dead animals
```

Comment:
@manuel489 I'm not sure why we have these `&#58`. Are they necessary?

Reply:
@ronaldtse I put them because there are cases where description items include colons:

![issue1_1](https://user-images.githubusercontent.com/33627611/151358945-b9e05cc7-d0c1-4af7-bdd8-0e2c96a0e515.PNG)


and other where they not:

![issue1_2](https://user-images.githubusercontent.com/33627611/151358976-7b22078b-bab5-46d8-be37-21b974d334e6.PNG)

Reply:
Should I remove these colon entities?